src/common/helper.js

```diff
@@ -1097,6 +1097,22 @@ async function getMemberDetailsByHandles (handles) {
   return _.get(res.body, 'result.content')
 }
 
+/**
+ * Get topcoder member details by handle.
+ *
+ * @param {Array} handle the user handle
+ * @returns {Array} the member details
```

Comment:
Params type `Array` doesn't match the actual data type.

Reply:
I have fixed this issue too.